src/index.css

```diff
@@ -43,9 +44,9 @@ code {
 
 nav {
   display: flex;
```

Comment:
- [ ] For better use of DRY rules you can group in one class rules that are repeated many times
like `display: flex` is repeated many times check your `css` file and try not to repeat yourself